R/DataMart.R

```diff
@@ -209,9 +209,10 @@ QLoadData <- function(filename, company.token = NA, ...)
     else if (res$status_code != 200)
     {
         if (!QFileExists(filename, show.warning = FALSE))
-            StopForUserError("The data file '", filename, "' does not exist in the Displayr cloud drive. ",
-                             "Ensure that the data file is in the Displayr cloud drive and its name has been correctly specified.",
-                             call. = FALSE)
+            StopForUserError(
+                "The data file '", filename, "' does not exist in the Displayr cloud drive. ",
+                "Ensure that the data file is in the Displayr cloud drive and its name has been correctly specified.",
+            )
```

Comment:
Just curious what the effect of this change is?

Reply:
The old call had the `call. = FALSE` argument in it. This throws a warning that `call.` argument is not used which is a nuisance/uninformative to a user.